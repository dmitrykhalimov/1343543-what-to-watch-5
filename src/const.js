export const ratingRanks = [
  {
    'Bad': 2.9
  },
  {
    'Normal': 4.9
  },
  {
    'Good': 7.9
  },
  {
    'Very good': 9.9
  },
  {
    'Awesome': 10
  }
];
