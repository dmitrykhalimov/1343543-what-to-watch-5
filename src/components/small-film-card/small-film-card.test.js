import React from "react";
import renderer from "react-test-renderer";
import SmallFilmCard from "./small-film-card";
import {MemoryRouter} from 'react-router-dom';
import {TEST_MOCKS} from "../../const";

describe(`SmallFilmCard`, () => {
  it(`Should SmallFilmCard render correctly`, () => {
    const tree = renderer
      .create(
          <MemoryRouter>
            <SmallFilmCard
              id={TEST_MOCKS.id}
              preview={TEST_MOCKS.preview}
              title={TEST_MOCKS.title}
              src={TEST_MOCKS.src}
            />
          </MemoryRouter>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
