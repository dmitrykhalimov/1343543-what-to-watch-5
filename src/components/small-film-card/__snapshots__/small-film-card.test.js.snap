// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallFilmCard Should SmallFilmCard render correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
>
  <a
    className="small-movie-card__link"
    href="/films/1"
    onClick={[Function]}
  >
    <div
      className="small-movie-card__image"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
    >
      <video
        autoPlay={false}
        height="175"
        muted={true}
        poster="https://thenewswheel.com/wp-content/uploads/2014/08/Interstate-60-review-BMW-convertible-James-Marsden-8-ball.jpg"
        src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <span
        className="small-movie-card__link"
      >
        
      </span>
    </h3>
  </a>
</article>
`;
